astronomer/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -65,6 +65,12 @@ async def get_client_async(self) -> AioBaseClient:
             self.log.info("Retrieving region_name from Connection.extra_config['region_name']")
             region_name = extra_config["region_name"]  # pragma: no cover
 
+        if "aws_session_token" in extra_config:
+            self.log.info(
```

Comment:
in the sync version, they also log `self.conn_repr` https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/utils/connection_wrapper.py#L349 
would it be possible to do similar logging here?

Reply:
whatever the previous async log was there I followed the same. If we want to follow the same in sync version I need to change all the log in this file

Reply:
As long we have a GitHub issue for it I'm ok.